react-design/my-app/src/PrevBtn.jsx

```diff
@@ -1,3 +1,7 @@
-export default function PrevBtn() {
-  return <button>Previous</button>;
+export default function PrevBtn({ items, current, set }) {
+  function handlePrev() {
```

Comment:
This logic works here! However, we passed 2 unnecessary props by doing so: `current` and `items` and by default, `set` as well. 

Instead I would do the logic in the parent `Rotating Banner` component just like how you did for `Indictators`, and pass this function as a prop instead! This way we only need one prop instead of three!